src/main/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/controller/DocumentController.kt

```diff
@@ -111,7 +132,12 @@ class DocumentController(
   @ApiResponses(
     ApiResponse(
       code = 400, message = "Bad request, e.g. virus scan returns error", response = ErrorResponse::class,
-      examples = Example(ExampleProperty(mediaType = "application/json", value = "{\n\"status\": 400,\n\"message\":\"VirusFoundException\"\n}"))
+      examples = Example(
+        ExampleProperty(
+          mediaType = "application/json",
```

Comment:
Future: I'd rather we used an imported constant everywhere for this and not a String literal, `"application/json"`.